packages/api/src/services/collateral-prices.ts

```diff
@@ -13,6 +13,18 @@ type Dependencies = {
   appClients: AppClients;
 };
 
+const tokenAddressesMap = {
+  uma: {
+    [CHAIN_IDs.mainnet]: "0x04Fa0d235C4abf4BcF4787aF4CF447DE572eF828",
+    [CHAIN_IDs.polygon_matic]: "0x3066818837c5e6eD6601bd5a91B0762877A6B731",
+  },
```

Comment:
this could be simplified a lot if we basically assume we are always mapping any contract address to mainnet from any chain. could just become a KV of [sidechainaddress]: [mainnetaddress]. i guess to be really bullet proof you might need to specify the from chain, in which case id probably have a different object for each chain to mainnet with the simple {[fromaddress]:toaddress} format